source/future.tex

```diff
@@ -115,7 +115,11 @@
 \begin{codeblock}
 int neck, tail;
 volatile int brachiosaur;
+brachiosaur = neck;             // OK, a volatile store
```

Comment:
"Ok" is an abbreviation of "Okay", not an acronym, so the "k" shouldn't be capitalized unless you intend the postal abbreviation for Oklahoma. Should these be "ok:" for consistency with "deprecated:"?

Reply:
"OK" is used 597 in the standard, and "Ok" 0 times :)

Reply:
Deviating from the "OK" convention is not in scope for this particular pull request. Please create a new editorial issue if you feel discussion is warranted.

Reply:
OK.

Reply:
"OK" is correct, it's not an abbreviation of okay. The "okay" spelling is a bacronym. "Ok" would be wrong.

Reply:
I am willing to allow that "OK" is as correct as "Ok" - style guides vary, and the word has no clear etymology. Let's consider this point resolved. I'm Ok with "OK".

Reply:
...although I do find the colon vs. comma inconsistency in "deprecated: " and "OK, " disturbing.